modules/core/arrow-test/src/main/kotlin/arrow/test/laws/InvariantLaws.kt

```diff
@@ -0,0 +1,44 @@
+package arrow.test.laws
+
+import arrow.Kind
+import arrow.core.andThen
+import arrow.core.identity
+import arrow.test.generators.genConstructor
+import arrow.test.generators.genFunctionAToB
+import arrow.typeclasses.Applicative
+import arrow.typeclasses.Eq
+import arrow.typeclasses.Invariant
+import io.kotlintest.properties.Gen
+import io.kotlintest.properties.forAll
+
+object InvariantLaws {
+
+  inline fun <F> laws(AP: Applicative<F>, EQ: Eq<Kind<F, Int>>): List<Law> =
```

Comment:
Inline isn't necessary. It's a leftover from before a refactor.

Reply:
Should I remove `inline` from the others tests too, including `noinline` for those who use it?
But leave `inline` for those tests with `reified`

Reply:
There should be no tests with reified AFAIK. Leave the fix for a different PR tho.